### ISSUE TITLE
Webauth / FIDO 2 HW Auth not working with brave browser

### ISSUE THREAD

**Issue by BlueCase** [bitwarden/clients#9081]
### Steps To Reproduce

1. Install and enable Bitwarden Extension
2. Try to login with Webauth/Fido2 Hardware Token 

### Expected Result

Pin/Key Dialog pop up

### Actual Result

Immediately following error appears.
![image](https://github.com/bitwarden/clients/assets/784375/31e9b613-a487-4450-9283-062e4489ba71)

### Additional Context

All working as expected with chromium. But since passkeys are supported following Window popup on fido2 auth.
![image](https://github.com/bitwarden/clients/assets/784375/bc7f6089-727a-4469-8e51-ebf5cb7a5c1b)
This window not pop up under brave.

Also opened a bug in the brave github project: https://github.com/brave/brave-browser/issues/38153

### Operating System

Linux

### Operating System Version

Manjaro

### Web Browser

Brave

### Browser Version

1.65.126 Chromium

### Build Version

124.0.6367.118

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by BlueCase** [comment]
Any update?

**Comment by quexten** [comment]
@BlueCase Could you confirm which packaging method you are using to install Brave? Aur (since it's Manjaro)? Flatpak?

**Comment by BlueCase** [comment]
@quexten 
I use the version from the manjaro extra repo
```
brave-browser  1.68.128-1 [Installiert]                                                                                                                 extra
    Web browser that blocks ads and trackers by default (binary release)
```

In generally it seems that all "Dialog Popup" actions from extensions not working.
With chromium from the same repository this things all working.

**Comment by quexten** [comment]
@BlueCase thanks for the extra info.
Just to confirm, with the bitwarden extension disabled, you can use fido2 on sites (with your security key)?

**Comment by BlueCase** [comment]
@quexten 
Yes or when I add manually the URLs to "blocked domain" list what I figured out in the meantime.

- Happen on all web pages where I tried to use fido2
- Login/Unlock also not shown when I trigger the unlock via the User/Password Field drop down.
![image](https://github.com/user-attachments/assets/54169059-0efd-4702-af88-26329ef8efe7)

**Comment by BlueCase** [comment]
@JeanLuX I don't think so..
Meanwhile it seems that all working normal. Also the related dialogs pop up again when you add a new passkey/fido2 login.
